.github/workflows/check-iOS11.yml

```diff
@@ -0,0 +1,12 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  check:
+    uses: ably/ably-cocoa/.github/workflows/check-base.yml@feature/1273-one-job-per-workflow
```

Comment:
I'm not feeling good about this? What will happen when we merge this PR and remove `feature/1273-one-job-per-workflow` branch?
It would be better if we use `composite action` for this type of workflow ([more here](https://damienaicheh.github.io/github/actions/2021/10/07/reuse-workflows-composite-action-en.html)) I know it's a bit more of work to do but IMO this is a more elegant solution.

Reply:
Sorry, I forgot to mention that after we agree upon this PR I'll change the branch into `main` there, but PR will fail and we must be landing it "as-is", because once it will be in the `main` itself it will work perfectly. WDYT @QuintinWillison

Reply:
In that case LGTM.